core/consensus/src/status.rs

```diff
@@ -67,45 +67,45 @@ impl StatusAgent {
 
 #[derive(Serialize, Deserialize, Clone, Debug, Display)]
 #[display(
-    fmt = "current_height {}, exec height {}, current_hash {:?}, latest_commited_state_root {:?} list state root {:?}, list receipt root {:?}, list confirm root {:?}, list cycle used {:?}, logs bloom {:?}",
+    fmt = "current_height {}, exec height {}, current_hash {:?}, latest_committed_state_root {:?} list state root {:?}, list receipt root {:?}, list confirm root {:?}, list cycle used {:?}, logs bloom {:?}",
     current_height,
```

Comment:
latest_committed_height?

Reply:
........double the ending constant letter 't' while change a present verb into past participle...
you type this wrong word everywhere and I correct them.....lol